third_party/lua/luadebug.lua

```diff
@@ -729,6 +740,7 @@ dbg = setmetatable({
         if condition then
             return
         end
+        verify_not_inside_of_console()
```

Comment:
Probably because of the fact one tried to enforce his personal tastes on some places where multiples variants could be employed for the same purpose. Personal taste is usually very subjective matter, and might be entirely ignored during review. 🤷‍♂️

Reply:
But in this case I've removed that extra function:
```diff
diff --git a/third_party/lua/luadebug.lua b/third_party/lua/luadebug.lua
index d0d557bb5..3fce9779b 100644
--- a/third_party/lua/luadebug.lua
+++ b/third_party/lua/luadebug.lua
@@ -707,16 +707,6 @@ repl = function(reason)
     until done
 end
 
---[[
-    Tarantool console keeps pointer to the console object
-    in the fiber.self().storage.console while inside of
-    REPL loop.
-]]
-local function verify_not_inside_of_console()
-    local fiber = require('fiber')
-    assert(fiber.self().storage.console == nil, DEBUGGER ..
-        ' is not yet compatible with interactive Tarantool console')
-end
 
 -- Make the debugger object callable like a function.
 dbg = setmetatable({
@@ -741,7 +731,15 @@ dbg = setmetatable({
         if condition then
             return
         end
-        verify_not_inside_of_console()
+
+        --[[
+            Prevent debugger from running from inside of Tarantool console.
+            Check pointer, which Tarantool console stored to the console object
+            in the fiber.self().storage.console while inside of
+            REPL loop.
+        --]]
+        assert(require('fiber').self().storage.console == nil, DEBUGGER ..
+            ' is not yet compatible with interactive Tarantool console')
 
         top_offset = top_offset or 0
         stack_inspect_offset = top_offset
```

Reply:
> Probably because of the fact one tried to enforce his personal tastes on some places where multiples variants could be employed for the same purpose. Personal taste is usually very subjective matter, and might be entirely ignored during review. 🤷‍♂️

And this is your "personal taste" and *personal opinion* ;) Thanks for the fix.